pkg/minikube/service/service.go

```diff
@@ -267,7 +267,7 @@ func PrintServiceList(writer io.Writer, data [][]string) {
 
 // WaitAndMaybeOpenService waits for a service, and opens it when running
 func WaitAndMaybeOpenService(api libmachine.API, namespace string, service string, urlTemplate *template.Template, urlMode bool, https bool,
```

Comment:
WaitAndMaybeOpenService already has too many arguments, let's not add another.

How about removing the `browser.OpenURL` call altogether, and leaving that up to `cmd/service.go` to handle. Then this function could be renamed to a more definite `WaitForService`, and be more testable.

Reply:
Agree with tstromberg

Reply:
Cool. Will make the changes